front/src/modules/ui/link/components/RoundedLink.tsx

```diff
@@ -21,14 +22,26 @@ const StyledClickable = styled.div`
   }
 `;
 
-export function RoundedLink({ children, href, onClick }: OwnProps) {
+export function RoundedLink({ children, href, onClick, type }: OwnProps) {
```

Comment:
let's pass label as a property and leave it to a SocialLink component to compute the displayValue
SocialLink => RoundedLink => Chip rounded